examples/nextjs/package.json

```diff
@@ -8,7 +8,7 @@
     "start": "next start"
   },
   "dependencies": {
-    "next": "10.x",
+    "next": "11.x",
```

Comment:
Should we make this `latest` instead?

Reply:
I think we left it pinned in case there was a breaking change 👍